08-CMSIS/core/inc/uart.h

```diff
@@ -0,0 +1,7 @@
+#ifndef __UART_H_
```

Comment:
USART should be initialized internally. We don't expect the exposure of UART information.